aeneas/src/main/CLOptions.v3

```diff
@@ -190,6 +190,8 @@ component CLOptions {
 		"Export the Wasm memory from generated modules.");
 	def WASM_PAGE_SIZE	= wasmOpt.newSizeOption("wasm.page-size", 65536,
 		"Specify the Wasm memory page size (must be a power of 2).");
+	def WASM_RESERVED_DATA_SIZE = wasmOpt.newSizeOption("wasm.reserved-data-size", 0,
```

Comment:
Just make it `reserved-data-size` and drop the `wasm.`; we might make use of this option on other platforms too.